### PR TITLE
feat(layout): convert chat layout to flexbox

### PR DESCRIPTION
**Pull request by jasonwoodland**
**What this PR does** 📖
- Update the chat layout to use flexbox

**Which issue(s) this PR fixes** 🔨
AP-1911

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: pages/chat/direct/Direct.less [hunk 1]
```diff
@@ -1,9 +1,6 @@
...
-  overflow: hidden;
-  position: relative;
-  display: flex;
-  flex-direction: column;
-  justify-content: flex-end;
+  margin: 0 @normal-spacing;
+  margin-top: auto;
+  overflow-y: auto;
+  overflow-x: hidden;
 }
```
Comment (josephmcg) [inline pages/chat/direct/Direct.less:6]: why are direct and group different? could we move these to a different place?

**Comment by molimauro** [comment]
Please make sure not to break chat scroll on Safari mobile 🙏  previously we were using flexbox in chat and had a lot of problems on mobile

**Review by josephmcg — APPROVED** [review]

**Comment by josephmcg** [comment]
actually, safari scroll is bugging out on ios simulator. it jumps whenever you reach the infinite scroll loading point

**Comment by jasonwoodland** [comment]
@josephmcg

> actually, safari scroll is bugging out on ios simulator. it jumps whenever you reach the infinite scroll loading point
I couldn't reproduce this bug, was it happening on dev too?

> also i can't swipe to go back while the blue, scroll to bottom banner is there
This is happening on dev too, if swiping horizontally on an `overflow: scroll` container whose scroll offset isn't at it's top/bottom, swiper will ignore the horizontal swipe gesture. I'll look into it some more

**Comment by WanderingHogan** [comment]
Closing PR, keeping branch, Jason is integrating some of these changes into other PR's related to chat

**Pull request closed without merging**